manager/orchestrator/global/global.go

```diff
@@ -156,12 +156,10 @@ func (g *Orchestrator) Run(ctx context.Context) error {
 				g.reconcileOneNode(ctx, v.Node)
 			case api.EventUpdateNode:
 				g.updateNode(v.Node)
-				switch v.Node.Status.State {
-				// NodeStatus_DISCONNECTED is a transient state, no need to make any change
-				case api.NodeStatus_DOWN:
+				// NodeStatus_DISCONNECTED is a transient state, no need to remove tasks
```

Comment:
Non-blocking nit:  `NodeStatus_DOWN` in this comment (I guess the name got changed at some point)